label_studio/users/views.py

```diff
@@ -142,7 +142,7 @@ def user_login(request):
 
 
 @login_required
-def user_account(request):
+def user_account(request, sub_path=None):
```

Comment:
it looks like a hack - provide at least a comment that it will be used as a redirect to a specific section like hotkeys